plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -124,7 +209,20 @@ def relative_path(self):
         related files get stored. For most unit types, this path is to one
         file.
         """
-        return self.id
+        return self.distribution_id
```

Comment:
Seems odd that an `distribution_id` is returned here.  But, based on the diff, I see it returned `id` previously so not changed by you.